changelogs/fragments/44561-vmware_host_config_manager-fix_key_error.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Fixed issue when a supported option isn't already set (https://github.com/ansible/ansible/issues/44561).
```

Comment:
```suggestion
- Fixed KeyError issue in vmware_host_config_manager when a supported option isn't already set (https://github.com/ansible/ansible/issues/44561).
```

Reply:
Since changelog entries will be combined together in final release version, there will be no clue about this line.